mlflow/R/mlflow/DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Type: Package
 Package: mlflow
 Title: Interface to 'MLflow'
-Version: 1.10.1
+Version: 1.12.1
```

Comment:
Skipped `1.11.0` on purpose (we just forgot to update when in the `mlflow 1.11` release)